test/index.ts

```diff
@@ -36,12 +41,11 @@ export function runTests(
 
     if (shouldRunTypeChecks === true) {
         describe(`type checks`, () => {
-            it('checks types', () => {
+            it('checks types', async () => {
+                const typeCheckCommand = `npm exec --offline tsc -- --noEmit --skipLibCheck`;
```

Comment:
Shouldn't we try to explicitly pass in the tsconfig.json of the project? Just like we do at the top level?

Reply:
I don't think so, I felt like passing in at the top level was a work around we had to do because of the extra files we wanted to ignore. I think this is a much better method under normal circumstances